crowbar_framework/lib/crowbar/lock.rb

```diff
@@ -16,33 +16,14 @@
 #
 
 module Crowbar
-  class Lock
-    attr_accessor :logger, :path, :name
-    attr_reader :path, :name, :file
+  module Lock
+    autoload :Base,
```

Comment:
We can drop 3 lines which need no regular maintenance and instead introduce 180 lines which do (as per #367)?

I'm not saying you're wrong, but I'd like to understand the reasoning behind this a bit better.